examples/trace-a-single-control/yaml/system-security-plan.yaml

```
@@ -0,0 +1,129 @@
+system-security-plan:
+  uuid: ##########ID##########
+
+  metadata:
+    title: Leveraging SaaS System Security Plan
+    last-modified: 2022-09-08T00:00:00.0000-04:00
+    version: "0.1"
+    oscal-version: 1.0.4
+    roles:
+      - id: admin
+        title: Administrator
+    parties:
+      - uuid: ##########ID##########
+        type: person
+  
+  import-profile:
+    href: profile.yaml
+  
+  system-characteristics:
+    system-ids:
+      - id: saas_system_iaas_customer
+    system-name: Leveraging SaaS System
+    description: >
+      NO CONTENT HERE FOR NOW
+    security-sensitivity-level: low
+    system-information:
+      information-types:
+        - title: System Assessment
+          description: This system handles development information pertaining to audit and assessment events to demonstrate the OSCAL workflow.
+          categorizations:
+            - system: https://doi.org/10.6028/NIST.SP.800-60v2r1
+              information-type-ids:
+                - C.3.5.1
+          confidentiality-impact:
+            base: fips-199-low
+          integrity-impact:
+            base: fips-199-low
```

I actually had to double check 800-60 Revision 2 and lazily looked up this PR to see what the structure was and translate it back into XML for another PR. FYI/FYE, the actually recommended (`base`) for `C.3.5.1` is Low-Moderate-Low.

```suggestion
            base: fips-199-moderate
```

If you wanted to override and say well the book says is moderate, but my AO, ISSO, organizational leadership, what have you, said it is different, you could do it.

```yaml
          integrity-impact:
            base: fips-199-moderate
            selected: fips-199-low
```